migrations/database_migration.py

```diff
@@ -0,0 +1,136 @@
+import json
```

Comment:
I suggest you to create folder `scripts ` and place script there. And please consider about giving this script more appropriate name (e.g. `transfed_db_data`).
Do not forget to create separate README in this folder and describe how to use your script